content/docs/setup/kubernetes/quick-start-gke-dm/index.md

```diff
@@ -208,19 +212,44 @@ http://localhost:9090/graph
 
 For more details, see [About the Prometheus Add-on](/docs/tasks/telemetry/querying-metrics/#about-the-prometheus-add-on).
 
-### ServiceGraph
+### Install mesh visualization tools
 
-Set up a tunnel to ServiceGraph:
+#### Kiali
 
-{{< text bash >}}
-$ kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=servicegraph -o jsonpath='{.items[0].metadata.name}') 8088:8088 &
-{{< /text >}}
+1.  Set up a tunnel to Kiali:
 
-You should see the Bookinfo service topology at
+    {{< text bash >}}
+    $ kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=kiali -o jsonpath='{.items[0].metadata.name}') 20001:20001 &
+    {{< /text >}}
 
-{{< text plain >}}
-http://localhost:8088/dotviz
-{{< /text >}}
+1.  You should see the Bookinfo service topology at
+
+    {{< text plain >}}
+    http://localhost:20001/kaili
+    {{< /text >}}
+
+The default username/password for the console is `admin`/`mysecret`.
+
+{{< image width="100%" ratio="53.33%"
+    link="./dm_kiali.png"
+    caption="Kiali"
+    >}}
+
+For more details, see [About the Kiali Add-on](/docs/tasks/telemetry/kiali/).
+
+#### ServiceGraph
+
+1.  Set up a tunnel to ServiceGraph:
```

Comment:
The TOC recommended a deprecation to servicegraph.  @douglas-reid may have more information.

Reply:
set defaults for `ServiceGraph` to false but left code in there to enable it as well as in the docs to state how to access its console